src/viking.js

```diff
@@ -1,11 +1,68 @@
 // Soldier
-class Soldier {}
-
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health = this.health - damage;
+  }
+}
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(name);
+    this.name = name;
+    this.health = health;
```

Comment:
Same as before comments, variables health and strength should not be declared as they are passed from parent.